reporting/impl/src/main/resources/reports/templates/application_list.ftl

```diff
@@ -38,12 +38,14 @@
                 <span class="points">${pointsFromAllTraversal}</span>
                 <span class="legend">story points</span>
             </div>
+            <#if (pointsFromSharedTraversal > 0) >
             <div class="effortPoints shared">
```

Comment:
yep, that is what I wanted, as keeping the space empty/not shifting looks weird IMO. I tried that approach